frontend/src/components/processManagement/ProcessModifyForm/index.tsx

```diff
@@ -0,0 +1,83 @@
+import Button from '@components/common/Button';
+import InputField from '@components/common/InputField';
+import TextField from '@components/common/TextField';
+import { Process } from '@customTypes/process';
+import React, { FormEvent, useState } from 'react';
+import { processMutaions } from '@hooks/process';
+import C from '../style';
+
+interface ProcessModifyFormProps {
+  process: Process;
+  isDeletable?: boolean;
+}
+
+export default function ProcessModifyForm({ process, isDeletable = false }: ProcessModifyFormProps) {
+  const { mutate: modifyMutate } = processMutaions.useModifyProcess();
+  const { mutate: deleteMutate } = processMutaions.useDeleteProcess();
+
+  const [formState, setFormState] = useState<Pick<Process, 'name' | 'description'>>({
+    name: process.name,
+    description: process.description,
+  });
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement> | React.ChangeEvent<HTMLTextAreaElement>) => {
+    const { name, value } = e.target;
+    setFormState((prev) => ({
+      ...prev,
+      [name]: value,
+    }));
+  };
+
+  const modifyProcess = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    modifyMutate({ ...formState, processId: process.processId });
+  };
+
+  const deleteProcess = () => {
+    if (window.confirm('정말로 삭제하시겠습니까?\n삭제하면 지원자들의 정보도 함께 삭제되며 복구할 수 없습니다.')) {
+      deleteMutate(process.processId);
+    }
+  };
+
+  return (
+    <C.ProcessForm onSubmit={modifyProcess}>
+      <InputField
+        label="프로세스 이름"
+        placeholder="32자 이내로 입력해주세요."
+        value={formState?.name}
```

Comment:
`formState`에 옵셔널 체이닝을 쓰셨군요. 이 경우 `formState`가 존재하지 않는다면 `undefined`가 들어가게 되겠네요. 또한 `formState`가 있더라도 `name` 프로퍼티가 없다면 마찬가지로 `undefined`가 들어갈 거에요.

이대로도 동작에 이상은 없겠지만, 가급적이면 예외적인 케이스에 대해 빈 문자열을 명시적으로 반환하는 것이 어떨지 제안드려 봅니다.
```suggestion
        value={formState?.name ?? ''}
```

Reply:
짚어주셔서 감사합니다~ 제안해주신대로 nullish 연산자를 사용해서 기본값을 주도록 수정할게요.

Reply:
아르 해당 내용 적용했습니다~ optional chaining이 불필요하게 들어간 부분도 있고, 필요한 부분은 초기값에 nullish를 적용하는 것으로 수정했어요.

- [커밋 바로가기](https://github.com/woowacourse-teams/2024-cruru/pull/137/commits/a6f12e9d085d6e775eb9db0b4f3f7de91b1c77f8)

Reply:
제가 제안드렸던 내용보다 더 좋은 것 같아요. 고생하셨습니다 :)